src/main/telemetry/crsf.c

```diff
@@ -337,20 +337,82 @@ void crsfFrameDeviceInfo(sbuf_t *dst) {
 }
 
 #if defined(USE_CRSF_CMS_TELEMETRY)
+#define CRSF_DISPLAYPORT_MAX_CHUNK_LENGTH   50
+#define CRSF_DISPLAYPORT_BATCH_MAX          0x3F
```

Comment:
What is the reason this was chosen?

Reply:
Max chunk length of 50 was mostly arbitrary, but it leaves a little space when we only have about 61ish bytes of encoded data.  This leaves room for extended header, subcommand, metadata, and sequence + a few bytes.

Reply:
0x3F was chosen to leave room for the prefix bytes 0x80 an 0x40 for first and last chunks.

Reply:
The max length could maybe be extended a few bytes, but not by much

Reply:
Ok, makes sense.